src/pages-and-resources/discussions/DiscussionAppList.jsx

```diff
@@ -1,71 +1,49 @@
-import React, { useState } from 'react';
-
+import React, { useCallback, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import { Button, Container, Row } from '@edx/paragon';
+import { useDispatch, useSelector } from 'react-redux';
+
 import messages from './messages';
 import DiscussionAppCard from './DiscussionAppCard';
 import FeaturesTable from './FeaturesTable';
+import { useModel, useModels } from '../../generic/model-store';
+import { fetchApps } from './data/thunks';
```

Comment:
The difference is that the lines with curlies are importing named exports, whereas anything being imported without curlies is importing the default from that file.  

If I had this file:

```
export default function blah() {
}

export const booyah = 12;
export const cackle = 'yes';
```

Then I would import those three exports as:

```
import blah, { booyah, cackle } from './thatFile';
```

Also note that `blah` on the import side can be named anything you want, which is just a weird idiosyncracy of default exports.  So I could also do:

```
import whatever, { booyah, cackle } from './thatFile';
```

And it would work, but be in bad taste.

Reply:
"in bad taste" love it